### DISCUSSION TITLE
Is there a list of key issues (issues) that must be focused and finalized for the release of a new stable version?

### DISCUSSION THREAD

**Discussion by scarletquasar** [sebastienros/jint#1211] [General]
I am currently using Jint in an [open source project](https://github.com/MelonRuntime/Melon) where I plan to promote scalability, so I would like to know that there is a line of things necessary to follow for the release of a new stable version. If so, I would like to know more about it and make myself available to make contributions.

**Comment by lahma** [comment]
By stable I presume you mean 3.0.0 without the preview label, or just next preview on NuGet? Basically the main readme states what's implemented and what's missing. Hard to say what's "must have" for the big release. One big concern that entire public API would need to be gone through and checked what makes sense and what not for normal use cases.

Also one should not be scared of the preview label as dependency, Jint has a lot of test coverage, but mostly lacking some features.

All contributions are always most welcome!

**Reply by scarletquasar** [reply]
Yes, a `3.0.0` without the preview label. 

> Also one should not be scared of the preview label as dependency, Jint has a lot of test coverage, but mostly lacking some features.

This makes me calmer on the subject, thanks for the information.

**Reply by lahma** [reply]
Yes,  some > 30 000 test cases run in both lax and strict mode for every build. All new development is mostly about getting new features implemented and passing more tests.

**Reply by scarletquasar** [reply]
Great, I have other question: there is any content to read about Jint? Currently I'm exploring by myself, there is any type of documentation?

**Reply by lahma** [reply]
There's the video and samples on main page, then blog posts with some web search.

**Reply by scarletquasar** [reply]
Oh good, do you mind if I want to write some docs? At least the basic. I would open pull requests to the repo, here

**Reply by lahma** [reply]
We are hoping to build a small documentation website. That would be ideal place to have them. All preparatory work like writing in tutorial and use case format would be greatly appreciated.
